__tests__/devenv-e2e/001-create.spec.js

```diff
@@ -111,22 +125,38 @@ describe( 'vip dev-env create', () => {
 
 		expect( await checkEnvExists( slug ) ).toBe( false );
 
-		const result = await cliTest.spawn( [
-			process.argv[ 0 ],
-			vipDevEnvCreate,
-			'--slug', slug,
-			'--app-code', 'image',
-			'--title', expectedTitle,
-			'--multisite', `${ expectedMultisite }`,
-			'--php', expectedPhpVersion,
-			'--wordpress', expectedWordPressVersion,
-			'--mu-plugins', 'image',
-			'-e', `${ expectedElasticsearch }`,
-			'-p', `${ expectedPhpMyAdmin }`,
-			'-x', `${ expectedXDebug }`,
-			'-A', `${ expectedMailpit }`,
-			'-H', `${ expectedPhoton }`,
-		], { env }, true );
+		const result = await cliTest.spawn(
+			[
+				process.argv[ 0 ],
+				vipDevEnvCreate,
+				'--slug',
+				slug,
+				'--app-code',
+				'image',
+				'--title',
+				expectedTitle,
+				'--multisite',
+				`${ expectedMultisite }`,
+				'--php',
+				expectedPhpVersion,
+				'--wordpress',
+				expectedWordPressVersion,
+				'--mu-plugins',
+				'image',
+				'-e',
+				`${ expectedElasticsearch }`,
+				'-p',
+				`${ expectedPhpMyAdmin }`,
+				'-x',
+				`${ expectedXDebug }`,
+				'-A',
+				`${ expectedMailpit }`,
+				'-H',
+				`${ expectedPhoton }`,
+			],
+			{ env },
+			true
+		);
```

Comment:
IMHO, the original version was more readable :-(

Reply:
Add exemptions for this and similar invocations with `// prettier-ignore`